IntegrationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -671,23 +671,6 @@ SWIFT_CLASS("_TtC10RevenueCat15ProductsManager")
 + (nonnull instancetype)new SWIFT_UNAVAILABLE_MSG("-init is unavailable");
 @end
 
-
-SWIFT_CLASS_NAMED("PromotionalOffer")
-@interface RCPromotionalOffer : NSObject
```

Comment:
Looking at this diff is super useful to see potentially non-backwards compatible changes. In this case I realized that #1141 didn't add a deprecation for `PromotionalOffer`. Let me send a PR for that.

Reply:
#1147

Reply:
great catch! I didn't think to do a deprecation since `PromotionalOffer` was only available on beta 9, but it can't hurt to have it there in case anyone was using that beta